README.md

```diff
@@ -59,6 +59,15 @@ Then access your server using a web browser on port 80. You will be asked for
 configuration information and redirected to a page where you can enter a
 transaction registration request.
 
+If your test page fails to redirect to your gateway URL, giving an error like:
+* Error: 4020 : Information received from an Invalid IP address.
+* Error: 4000 : The VendorName is invalid or the account is not active.
+then you have likely not configured either the vendor name or the valid IPs correctly. Note that despite the precision of the exception messages, either message could indicate either case.
```

Comment:
Markdown formatting error, needs blank line to separate it from the bullet list.